codegen/constants.yaml

```diff
@@ -159,6 +163,48 @@ groups:
           - name: METHOD_RESPONSE_STATUS_CODE
             value: "methodResponseStatusCode"
             desc: "Status code sent with method response"
+      - name: GetDigitalTwinC2d
```

Comment:
Aren't we using "Plug and Play" instead of "Digital Twins"? 

IMO we should avoid using digital twins in our naming here because it can get confused with the Azure Digital Twins service.